libs/custom-params-component/custom-params-component.js

```diff
@@ -0,0 +1,79 @@
+export const breakpoints = {
+  s: 0,
+  m: 768,
+  l: 1024,
+  xl: 1280,
+  xxl: 1920,
```

Comment:
for the viewports I think we should go with the Adobe "standard" of 600, 900, 1200, 1500